app/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
     id("com.google.devtools.ksp")
     id("com.google.gms.google-services")
     id("com.google.firebase.crashlytics")
+    id("io.gitlab.arturbosch.detekt")
```

Comment:
Why do we need this?

Reply:
There was an error that your app folder didn't have detekt

Reply:
Got it, that's fine. Let's keep it